backend/app/tests/api/routes/test_prompts.py

```
@@ -40,364 +42,686 @@ def mock_current_user_fixture():
 @patch("app.models.PromptVersion")
 @patch("app.models.Prompt")
-def test_create_prompt_success_no_tags(MPr: MagicMock, MPV: MagicMock, client: TestClient, mock_db_session_fixture: MagicMock, mock_current_user_fixture: MagicMock):
-    p_in = PromptCreate(name="Test", content="Content", type="simple", visibility="public")
-    p_mock = MagicMock(spec=Prompt, id=uuid.uuid4(), tags=[]); MPr.return_value=p_mock
-    for k,v in p_in.model_dump(exclude={"tag_ids"}).items(): setattr(p_mock,k,v)
-    p_mock.created_by = mock_current_user_fixture.id; p_mock.content = p_in.content; p_mock.input_vars = p_in.input_vars
+def test_create_prompt_success_no_tags(
+    MPr: MagicMock,
+    MPV: MagicMock,
+    client: TestClient,
+    mock_db_session_fixture: MagicMock,
```

Ruff (0.11.9)

Unused function argument: `mock_db_session_fixture`

(ARG001)